src/app/pipes/project-region-filter.pipe.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { ProjectRegionFilterPipe } from './project-region-filter.pipe';
+import { Project } from '../models/project';
+
+describe('ProjectRegionPipe', () => {
+
+  const string = 'Peace';
+  let value: Project[];
+  let expectedResponse: Project[];
+
+  const pipe = new ProjectRegionFilterPipe();
+
+  describe('given a valid response', () => {
+    it('returns 1 item', () => {
+      value = [new Project({region: 'Peace'})];
+      expectedResponse = [new Project({region: 'Peace'})];
+
+      expect(pipe.transform(value, string).length).toBe(expectedResponse.length);
+    });
+    it('returns n items', () => {
+      value = [
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'})
+      ];
+      expectedResponse = [
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'})
+      ];
+
+      expect(pipe.transform(value, string).length).toBe(expectedResponse.length);
+    });
+  });
+  describe('given a mix of different regions', () => {
+    beforeEach(() => {
+      value = [
+        new Project({region: 'Peace'}),
+        new Project({region: 'Lower Mainland'}),
+        new Project({region: 'Peace'}),
+        new Project({region: 'Lower Mainland'})
+      ];
+      expectedResponse = [
+        new Project({region: 'Peace'}),
+        new Project({region: 'Peace'}),
+      ];
+    });
+    it('returns two items', () => {
```

Comment:
Please update the `it` description with something more explanatory (e.g.: 'only returns the matching items').
You could also consolidate all the assertions inside the same *it*, rather than having two since they are all used to verify only the matching items were returned.